src/MainWindow.cpp

```diff
@@ -154,10 +153,15 @@ MainWindow::MainWindow(const QString profile, QWidget *parent)
         trayIcon_->setVisible(userSettings_->tray());
 
         if (hasActiveUser()) {
-                QString token       = settings.value("auth/access_token").toString();
-                QString home_server = settings.value("auth/home_server").toString();
-                QString user_id     = settings.value("auth/user_id").toString();
-                QString device_id   = settings.value("auth/device_id").toString();
+                UserSettings us;
+                QString prefix;
+                if (us.profile() != "")
```

Comment:
Why is accessing userSettings_ not enough?

Reply:
That's a good point.